bundle/manifests/instrumentation.opentelemetry.io_instrumentations.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.6.0-beta.0
   creationTimestamp: null
-  name: instrumentations.opentelemetry.io
+  name: instrumentations.instrumentation.opentelemetry.io
```

Comment:
No, this is too verbose. It should really be `instrumentations.opentelemetry.io`.